otx/algorithms/detection/tasks/dataset/__init__.py

```diff
@@ -0,0 +1,9 @@
+"""OTX Algorithms - Detection Dataset."""
```

Comment:
I don't know if it's proper to put dataset-related codes in tasks folder. In anomaly, task folder is for such type as train/inference/nncf. some modules like model, datasets were in adapter folder. In my case, I made new subfolder called extension like `otx/algorithms/classification/extension`.

Reply:
Okay. Since we are just in initial refactoring phase, let's take dataset in auxiliary path and move on to adapter folder.

Reply:
+ Maybe the codes included in utils can be modified to belong to adapters. (If the adapters are implemented, I will consider it.)

Reply:
@supersoob Could you please check this one more time?